features/bootstrap/JoindInContext.php

```diff
@@ -84,4 +84,14 @@ protected function getService(string $name)
     {
         return $this->kernel->getContainer()->get($name);
     }
+
+    /**
+     * @When I fetch all meetups with talks and their comments from Joindin in one go
+     */
+    public function iFetchAllMeetupsWithTalksAndTheirCommentsFromJoindIn()
```

Comment:
@msvrtan - I'm reusing existing step implementations here, because there's no way to test the actual call to code without exposing HomepageController:FetchJoindInData as a service. That's what I meant when I said it felt 'hacky'.
If this is ok with you, I'll merge it.